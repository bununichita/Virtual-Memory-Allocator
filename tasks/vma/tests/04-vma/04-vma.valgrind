==36227== Memcheck, a memory error detector
==36227== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==36227== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==36227== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==36227== Parent PID: 36226
==36227== 
==36227== 
==36227== HEAP SUMMARY:
==36227==     in use at exit: 0 bytes in 0 blocks
==36227==   total heap usage: 13 allocs, 13 frees, 8,434 bytes allocated
==36227== 
==36227== All heap blocks were freed -- no leaks are possible
==36227== 
==36227== For lists of detected and suppressed errors, rerun with: -s
==36227== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
