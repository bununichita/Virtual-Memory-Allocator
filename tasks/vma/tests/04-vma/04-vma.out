Total memory: A bytes
Free memory: 8 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 2

Block 1 begin
Zone: 1 - 3
Miniblock 1:	1	-	2	| RW-
Miniblock 2:	2	-	3	| RW-
Block 1 end
